### PR TITLE
small site fixes

### PR DESCRIPTION
**Pull request by martinraison**
@2is10 @stkem a few small fixes, including replacing /about_us with /about

### CODE REVIEW

**Comment by 2is10** [comment]
thanks!

File: kifi-backend/modules/shoebox/conf/shoebox.routes [hunk 1]
```diff
@@ -678,5 +676,6 @@ GET     /ng                        com.keepit.controllers.website.AngularDistAss
 GET     /dist/*file                com.keepit.controllers.website.AngularDistAssets.at(path="/angular/dist", file)
 GET     /img/*file                 com.keepit.controllers.website.AngularImgAssets.at(path="/angular/img", file)
 
+GET     /about/*file               com.keepit.controllers.website.AboutAssets.at(path="/public/about_us", file)
```
Comment (2is10) [inline kifi-backend/modules/shoebox/conf/shoebox.routes:679]: Would it be possible in `AboutAssets` to add a `.html` extension if `file` has no extension? that way we could serve our pages without it. If so, we should probably also 301 redirect `.html` requests there, or serve 404s for them.

**Comment by 2is10** [comment]
Did you consider hosting these static assets at the nginx level? That way we could update them without doing a shoebox push, and they’d still be available even if shoebox became unavailable for whatever reason.

File: kifi-backend/modules/shoebox/conf/shoebox.routes (hunk 1 above)
Comment (martinraison) [inline kifi-backend/modules/shoebox/conf/shoebox.routes:679]: Indeed, this should be fairly easy to do.

**Comment by martinraison** [comment]
I guess this is possible - I have never touched the nginx config until now so I may try someday. I have a few higher priority tasks to deal with first, though.

**Comment by 2is10** [comment]
Yeah, certainly not critical, and there are pros and cons of both
approaches.​

**Pull request merged**